ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockTransactionSelector.java

```diff
@@ -210,6 +213,16 @@ private TransactionSelectionResult evaluateTransaction(
       return TransactionSelectionResult.DELETE_TRANSACTION_AND_CONTINUE;
     }
 
+    if (processableBlockHeader.getBaseFee().isPresent()
+        && Wei.of(processableBlockHeader.getBaseFee().get())
+                .compareTo(actualMinTransactionGasPriceInBlock)
+            > 0) {
+      LOG.error(
+          "transaction price must be greater than base fee, dropping transaction {} from the pool",
+          transaction.getHash().toHexString());
+      return TransactionSelectionResult.DELETE_TRANSACTION_AND_CONTINUE;
```

Comment:
Can we just `CONTINUE` here then? We already remove the lowest priority transaction in `addTransaction` in `PendingTransactions` so when the 1559 tx pool prioritization lands, that will automatically be taken care of.